revolv/templates/project/edit_project.html

```diff
@@ -132,4 +145,28 @@
         });
     });
 </script>
+<!-- Initializes the category selection tool -->
+<script src="{% static 'chosen/chosen.jquery.js' %}"></script>
+<script src="{% static 'chosen/docsupport/prism.js' %}" type="text/javascript" charset="utf-8"></script>
+<script>
+    $( document ).ready(function() {
+        // populates the currently selected categories if none of them have been selected
+        {% if project.id %}
+        var current_categories = "{{current_categories}}";
+        current_categories = current_categories.split("&#39;").join("");
+        current_categories = current_categories.split(" ").join("");
+        current_categories = current_categories.substring(1, current_categories.length - 1);
+        $('#id_categories_list').val(current_categories);
+        {% endif %}
+
+        // initializes chosen
+        $(".chosen-select").chosen();
+        // sets the on change listener to update the hidden input field whenever categories are updated
```

Comment:
Why did you choose to make it a hidden element? You could just do `<select multiple>` right?